pandas/tests/io/excel/test_writers.py

```diff
@@ -224,7 +224,7 @@ def test_read_excel_parse_dates(self, ext):
 class _WriterBase:
```

Comment:
Doesn't need to be done here but we can probably get rid of this class altogether since it only contains the fixture. I also think it is better to use partials instead of setting global settings (we do this on the reading side) so something to look at in the future if you are interested in closing #27096